### ISSUE TITLE
proto-dep does not support repos with any group depth

### ISSUE THREAD

**Issue by seanamos** [stormcat24/protodep#95]
While github.com only has repos (for now) with the following structure: `github.com/{owner}/{repo}.git`, other hosting services (gitlab.com) can have repos with the following structure: `gitlab.com/a/b/c/d/e/f.git` (subgroups).

When trying to use a repo like: `gitlab.com/a/b/c/d` as a target, proto-dep will try connect to `ssh://gitlab.com/a/b.git`.

The quick fix is to change `pkg/config/schema.go/Repository()` to return the target as specified. Testing locally, this just works. However, this is technically a breaking change.

**Comment by stormcat24** [comment]
@seanamos Would you please send a pull request to this repository? I want to confirm and consider it.

**Comment by belenkoArrival** [comment]
Is this still a problem? Currently trying to use subgroup paths (gitlab) and it seems to have the same issue.

**Comment by seanamos** [comment]
@belenkoArrival Yes the problem still exists. I apologize, I haven't had time to push up a PR with the fix (which is quite simple), my youngest son was born just a few days ago so I simply haven't had time.

**Comment by seosite** [comment]
look forward to solving this problem at an early date...

**Comment by hr3685930** [comment]
look forward to solving this problem at an early date...
